site/content/en/docs/drivers/docker.md

```diff
@@ -33,3 +33,11 @@ The Docker driver allows you to install Kubernetes into an existing Docker insta
 - On macOS or Windows, you may need to restart Docker for Desktop if a command gets hung
 
 - Run `--alsologtostderr -v=1` for extra debugging information
+
+- On Linux, if you want to run MySQL pod, you need to disable AppArmor for mysql profile
+
+   AppArmor is access control security system for Linux. This is enabled on Ubuntu by default.
```

Comment:
I suggest instead of giving them the insrtuctions, link them to the issue, /
just say running mysql in privileged mode has an issue on docker driver, there is a workarround on docker site, please check this link. 

since this is somethign related to security, we shouldnt advise them to lower their security level, the user needs to study and take an educated risk

Reply:
Updated. 

> since this is somethign related to security, we shouldnt advise them to lower their security level, the user needs to study and take an educated risk

Thank you for precise advice! I couldn't consider about those risks.